docs/conf/vilnius/2019/news/announcing-talks.rst

```diff
@@ -0,0 +1,57 @@
+:template: {{year}}/generic.html
+
+.. post:: May 16, 2019
+   :tags: vilnius-2019, speakers, tickets, visiting
+
+Announcing Speakers and Talks
+=============================
+
+Greetings, fellow documentarians! We're writing today with an update on the fast-approaching Vilnius event, and we're very excited to announce the speakers and talks for the event.
+
+Thank you so much to everyone who took the time to send us a pitch – we literally wouldn't have an event without you!
+
+We also have some info about ticket purchasing (now's the time!), trip planning (Vilnius is beautiful!), and Writing Day (start thinking about projects!).
+
+Read on for more details!
```

Comment:
I'd leave this out considering it's a fairly short post and the previous paragram is already exclamation-heavy. Don't care strongly at all.